webpack.config.js

```diff
@@ -65,11 +65,28 @@ const webpackConfig = {
 			{
 				include: /node_modules\/tinymce/,
 				loader: 'imports?this=>window',
-			}
+			},
+			{
+				test: /\.scss$/,
+				loaders: [
+					'isomorphic-style-loader',
+					'css-loader?camelCase=dashes&importLoaders=1',
+					'sass-loader',
+					"webpack-append?@import 'assets/stylesheets/shared/component';",
+				]
+			},
+			{
+				test: /\/client\/components\/vertical-nav.+index\.jsx$/,
+				loaders: [
+					'imports?style=./style.scss,withStyles=isomorphic-style-loader/lib/withStyles',
+					'exports?default=withStyles.default( style )( exports.default )'
+				]
+			},
+
 		]
 	},
 	resolve: {
-		extensions: [ '', '.json', '.js', '.jsx' ],
+		extensions: [ '', '.json', '.js', '.jsx', '.scss' ],
```

Comment:
Do we need to resolve the extension if we're not expecting the SASS files to be imported by developers? IIRC, this is the difference between allowing:

```js
import './style.scss';
```

And:

```js
import './style';
```

(Latter with `extensions` defined)